client/sites/overview/components/migration-overview/components/migration-started-difm.tsx

```diff
@@ -1,7 +1,32 @@
 import { globe, group, Icon, scheduled, envelope } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
+import { ReactElement } from 'react';
 import { Container, Header } from './layout';
 
+type MigrationStartedListProps = {
+	children: ReactElement< MigrationStartedItemProps > | ReactElement< MigrationStartedItemProps >[];
+};
+
+type IconType = Parameters< typeof Icon >[ 0 ][ 'icon' ];
+
+type MigrationStartedItemProps = {
+	icon: IconType;
```

Comment:
Minor non-blocking nit: we could also inline the `icon` definition. The `IconType` is only used in this one location, and it's tricky to name it correctly.
```suggestion
type MigrationStartedItemProps = {
	icon: Parameters< typeof Icon >[ 0 ][ 'icon' ];
```

Reply:
That's a good idea. I'll do that!